ui/app/utils/resolve.server.ts

```diff
@@ -1,3 +1,9 @@
+/**
+ * Server-side utilities for resolving file content for inputs and model inferences.
+ *
+ * TODO(shuyangli): find a nice way to avoid duplicating all the types with resolved files; possibly lazily load them.
+ */
```

Comment:
https://github.com/tensorzero/tensorzero/issues/4675

Reply:
I'm not 100% convinced by that (loading files at the edge has nice properties) but we can say this is the default solution for now